lib/Translation/Translator.php

```diff
@@ -24,10 +24,16 @@
 use Symfony\Component\Translation\MessageCatalogue;
 use Symfony\Component\Translation\MessageCatalogueInterface;
 use Symfony\Component\Translation\TranslatorBagInterface;
-use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Symfony\Contracts\Translation\LocaleAwareInterface;
+use Symfony\Contracts\Translation\TranslatorTrait;
 
-class Translator implements TranslatorInterface, TranslatorBagInterface
+class Translator implements TranslatorInterface, TranslatorBagInterface, LocaleAwareInterface
```

Comment:
We should also additionally implement the legacy interface `Symfony\Component\Translation\TranslatorInterface` here, same as Symfony does, see: 
https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Translation/Translator.php#L32

Otherwise it would be a proper BC break :)